redis-cluster/README.md

```diff
@@ -0,0 +1,121 @@
+# redis cluster base manifests
```

Comment:
Would you have anything against me changing the README structure to the same as the other manifests from this repository? I would use separate PR for that

Reply:
Nope thats fine

Reply:
Btw, did you sourced this manifest from any particular place, or is it solely result of your experiments?

Reply:
Mainly experiments although I did look at a few different approaches one being bitnami.